tachyon/math/finite_fields/prime_field_fallback.h

```diff
@@ -295,20 +299,18 @@ class PrimeField<_Config, std::enable_if_t<!_Config::kUseAsm &&
   // MultiplicativeGroup methods
   constexpr std::optional<PrimeField> Inverse() const {
     PrimeField ret{};
-    if (LIKELY(value_.template MontgomeryInverse<Config::kModulusHasSpareBit>(
-            Config::kModulus, Config::kMontgomeryR2, ret.value_))) {
+    if (inverter.Invert(value_, ret.value_)) {
       return ret;
     }
-    LOG_IF_NOT_GPU(ERROR) << "Inverse of zero attempted";
+    LOG_IF_NOT_GPU(ERROR) << "Inversion of zero attempted";
```

Comment:
Please restore this change! there's another place to use this error message, it's better to add this change in a separate PR if it is needed.